### ISSUE TITLE
Getting Null pointer on Taplytics.resetAppUser

### ISSUE THREAD

**Issue by hetang-shah** [taplytics/Taplytics-Android-SDK#30]
Getting Null pointer on debug build. Release build is fine.
```
 java.lang.NullPointerException: Attempt to invoke virtual method 'void com.taplytics.dog.aardvark(java.util.Map, com.taplytics.dog$alpaca)' on a null object reference
                                                                     at com.taplytics.crane.aardvark(SourceFile:1191)
                                                                     at com.taplytics.ant.resetAppUser(SourceFile:500)
                                                                     at com.taplytics.sdk.Taplytics$4.run(SourceFile:432)
                                                                     at com.taplytics.crane$aardvark.execute(SourceFile:1976)
                                                                     at com.taplytics.crane.aardvark(SourceFile:1915)
                                                                     at com.taplytics.sdk.Taplytics.resetAppUser(SourceFile:428)
                                                                     at com.fivestars.mma.data.tracking.impl.TaplyticsImpl.trackLogout(TaplyticsImpl.kt:66)
```

SDK version: 1.17.1
Platform: Android Native

**Comment by VicV** [comment]
Hey @hetang-shah, sorry for the delayed response.

Thank you for the information. I'll be looking into this shortly.

**Comment by VicV** [comment]
Late to close, but this was fixed in [1.18.1](https://github.com/taplytics/Taplytics-Android-SDK/releases/tag/1.18.1)

Thanks for reporting this!
